docs/index.rst

```diff
@@ -0,0 +1,32 @@
+.. Reading List documentation master file, created by
+   sphinx-quickstart on Mon Feb  2 15:08:06 2015.
+   You can adapt this file completely to your liking, but it should at least
+   contain the root `toctree` directive.
+
+Welcome to Reading List's documentation!
```

Comment:
Anyway, "welcome to" seems weird to me. I believe we should remove the "welcome" part.